### ISSUE TITLE
Incompability with blocks from builders at pagination to top, and cannot duplicate the pagiantion block at the recent updates.

### ISSUE THREAD

**Issue by gfarma** [woothemes/woocommerce#47717]
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Hello, 

1) At one of the recent updates the ability to duplicate the pagination block is not availabe ( I use 8.7.0 and I it is not possible)
2) Also after the update 8.7.0 if you add a another block inside the product collection query block, the interactivity pagination with back to top is not working. (even if you add a simple text block from a FSE builder - not core). 

### Expected behavior

We should have the ability to duplicate the pagination block (at long pages it is nice to have both bottom and top the pagination)

### Actual behavior

1) It doesn't duplicate it.
2) we should be able to add blocks out of core, to extend the features.

### Steps to reproduce

1) try to copy pate the pagination block, or duplicate it at version 8.9.1
2) try to add a block for other author inside the product collection block

### WordPress Environment

`
### WordPress Environment ###

WordPress address (URL): https://fshome.gr
Site address (URL): https://fshome.gr
WC Version: 8.7.0
REST API Version: ✔ 8.7.0
Action Scheduler Version: ✔ 3.7.2
Log Directory Writable: ✔
WP Version: 6.5.3
WP Multisite: –
WP Memory Limit: 1 GB
WP Debug Mode: –
WP Cron: ✔
Language: el
External object cache: –

### Server Environment ###

Server Info: fastpath-nginx
PHP Version: 8.1.28
PHP Post Max Size: 100 MB
PHP Time Limit: 360
PHP Max Input Vars: 9000
cURL Version: 7.88.1
OpenSSL/3.0.2

SUHOSIN Installed: –
MySQL Version: 10.6.18-MariaDB-ubu2204
Max Upload Size: 100 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 8.8.2
WC Database Prefix: wp_
Total Database Size: 3184.19MB
Database Data Size: 1977.53MB
Database Index Size: 1206.66MB
wp_woocommerce_sessions: Data: 1.11MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 5.52MB + Index: 3.03MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 96.63MB + Index: 19.06MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 10.52MB + Index: 8.47MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 9.52MB + Index: 4.03MB + Engine InnoDB
wp_adtribes_my_conversions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_aioseo_cache: Data: 0.03MB + Index: 0.03MB + Engine InnoDB
wp_aioseo_crawl_cleanup_blocked_args: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_aioseo_crawl_cleanup_logs: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_aioseo_notifications: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_aioseo_posts: Data: 0.14MB + Index: 0.02MB + Engine InnoDB
wp_berocket_termmeta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_blc_filters: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_blc_instances: Data: 0.05MB + Index: 0.05MB + Engine InnoDB
wp_blc_links: Data: 0.13MB + Index: 0.11MB + Engine InnoDB
wp_blc_synch: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_bv_activities_store: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_cardlink_gateway_transactions: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_cmplz_cookiebanners: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_cmplz_cookies: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_cmplz_dnsmpd: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_cmplz_services: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_commentmeta: Data: 1.52MB + Index: 0.84MB + Engine InnoDB
wp_comments: Data: 17.55MB + Index: 10.61MB + Engine InnoDB
wp_cp_calculated_fields_form_discount_codes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_cp_calculated_fields_form_posts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_cp_calculated_fields_form_revision: Data: 0.27MB + Index: 0.00MB + Engine InnoDB
wp_cp_calculated_fields_form_settings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_eurobank_transactions: Data: 0.17MB + Index: 0.00MB + Engine InnoDB
wp_e_events: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_e_notes: Data: 0.02MB + Index: 0.17MB + Engine InnoDB
wp_e_notes_users_relations: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_e_submissions: Data: 0.28MB + Index: 0.44MB + Engine InnoDB
wp_e_submissions_actions_log: Data: 0.06MB + Index: 0.11MB + Engine InnoDB
wp_e_submissions_values: Data: 1.52MB + Index: 0.16MB + Engine InnoDB
wp_failed_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_fww_object_ancestors: Data: 35.56MB + Index: 85.58MB + Engine InnoDB
wp_itsec_lockouts: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_itsec_logs: Data: 0.09MB + Index: 0.09MB + Engine InnoDB
wp_itsec_temp: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_iwp_sessions: Data: 0.30MB + Index: 0.00MB + Engine InnoDB
wp_jetpack_sync_queue: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_jet_widgets_styles: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailchimp_carts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_options: Data: 83.28MB + Index: 15.72MB + Engine InnoDB
wp_pmxe_exports: Data: 1.50MB + Index: 0.00MB + Engine InnoDB
wp_pmxe_google_cats: Data: 0.38MB + Index: 0.00MB + Engine InnoDB
wp_pmxe_posts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxe_templates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_files: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_hash: Data: 0.13MB + Index: 0.08MB + Engine InnoDB
wp_pmxi_history: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_images: Data: 1.52MB + Index: 0.88MB + Engine InnoDB
wp_pmxi_imports: Data: 6.52MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_posts: Data: 0.47MB + Index: 0.45MB + Engine InnoDB
wp_pmxi_templates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_postmeta: Data: 589.98MB + Index: 724.16MB + Engine InnoDB
wp_posts: Data: 399.66MB + Index: 100.16MB + Engine InnoDB
wp_queue: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_say_what_strings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_scalability_pro_cache: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_scalability_pro_post_count_cache: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_searchwp_index: Data: 11.52MB + Index: 17.06MB + Engine InnoDB
wp_searchwp_log: Data: 42.56MB + Index: 47.63MB + Engine InnoDB
wp_searchwp_status: Data: 1.52MB + Index: 1.00MB + Engine InnoDB
wp_searchwp_tokens: Data: 1.52MB + Index: 2.55MB + Engine InnoDB
wp_search_filter_fieldmeta: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_search_filter_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_search_filter_options: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_search_filter_queries: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_search_filter_querymeta: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_search_filter_stylemeta: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_search_filter_styles: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_snippets: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_splitit_logs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_spro_slow_query_log: Data: 87.59MB + Index: 0.00MB + Engine InnoDB
wp_sss_post_titles: Data: 2.52MB + Index: 0.00MB + Engine InnoDB
wp_sss_tokens: Data: 33.55MB + Index: 12.55MB + Engine InnoDB
wp_superspeedysearch: Data: 425.69MB + Index: 41.83MB + Engine InnoDB
wp_superspeedysearch_suggestions: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_superspeedysearch_suggestions_useradded: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_superspeedysearch_suggestions_userremoved: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_superspeedysearch_terms: Data: 1.52MB + Index: 0.09MB + Engine InnoDB
wp_termmeta: Data: 1.52MB + Index: 0.55MB + Engine InnoDB
wp_terms: Data: 0.33MB + Index: 0.45MB + Engine InnoDB
wp_term_relationships: Data: 14.55MB + Index: 17.03MB + Engine InnoDB
wp_term_taxonomy: Data: 0.41MB + Index: 0.91MB + Engine InnoDB
wp_tm_taskmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_tm_tasks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_userfeedback_surveys: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_userfeedback_survey_responses: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_usermeta: Data: 4.52MB + Index: 7.06MB + Engine InnoDB
wp_users: Data: 0.17MB + Index: 0.30MB + Engine InnoDB
wp_wccs_conditions: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wccs_condition_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wcpdf_invoice_number: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 1.52MB + Index: 0.47MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_orders: Data: 4.52MB + Index: 2.39MB + Engine InnoDB
wp_wc_orders_meta: Data: 14.52MB + Index: 22.16MB + Engine InnoDB
wp_wc_order_addresses: Data: 4.52MB + Index: 3.97MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_operational_data: Data: 2.52MB + Index: 0.69MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 3.52MB + Index: 1.59MB + Engine InnoDB
wp_wc_order_stats: Data: 1.52MB + Index: 0.78MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 14.56MB + Index: 10.55MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 8.52MB + Index: 14.61MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wpfm_backup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpforms_logs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpforms_payments: Data: 0.02MB + Index: 0.14MB + Engine InnoDB
wp_wpforms_payment_meta: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wpforms_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpgb_cards: Data: 0.27MB + Index: 0.03MB + Engine InnoDB
wp_wpgb_facets: Data: 0.09MB + Index: 0.05MB + Engine InnoDB
wp_wpgb_grids: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wpgb_index: Data: 25.55MB + Index: 24.58MB + Engine InnoDB
wp_wpi_scheduler_lock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wplc_webhooks: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpmailsmtp_debug_events: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpmailsmtp_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpmerge_options: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

acf-field: 37
acf-field-group: 7
afreg_fields: 2
attachment: 48466
br_filters_group: 2
br_product_filter: 60
custom_css: 2
def_reg_fields: 10
elementor-hf: 1
elementor_icons: 1
elementor_library: 77
ep-synonym: 1
gm_menu_block: 1
iwp-importer: 3
jet-menu: 4
mgmlp_media_folder: 80
nav_menu_item: 111
oembed_cache: 11
page: 23
post: 2
po_product_options: 14
product: 16619
product_variation: 35979
pwbe_filter: 7
pwbf_event: 1
revision: 32951
rmp_menu: 1
schema: 3
shop_coupon: 2
shop_order: 11007
shop_order_refund: 68
tm_global_cp: 32
wpephpcompat_jobs: 41
wpforms: 3
wpmm_theme: 4
wp_block: 5
wp_global_styles: 2
wp_navigation: 1
wp_template: 11
wp_template_part: 5

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (72) ###

WP Sheet Editor - WooCommerce Products (Premium): by WP Sheet Editor – 1.8.14-beta.2
Add From Server: by Dion Hulse – 3.4.5
Advanced Cron Manager: by BracketSpace – 2.5.5
Advanced Custom Fields: by WP Engine – 6.3.0
The SEO Framework: by The SEO Framework Team – 5.0.6
Autoptimize: by Frank Goossens (futtta) – 3.1.11
Breadcrumb Block: by Phi Phan – 1.0.14
Cardlink Payment Gateway: by Cardlink – 1.0.7
Check Conflicts: by ioannup – 1.1.6
Classic Widgets: by WordPress Contributors – 0.3
Code Snippets Pro (Premium): by Code Snippets Pro – 3.6.4
Code Snippets: by Code Snippets Pro – 3.6.4
Complianz | GDPR/CCPA Cookie Consent: by Really Simple Plugins – 7.0.5
Facebook for WooCommerce: by Facebook – 3.2.2
Flexible Shipping PRO: by Octolize – 2.18.0
Flexible Shipping: by Octolize – 4.25.0
GA Google Analytics: by Jeff Starr – 20240308
Site Kit by Google: by Google – 1.127.0
GreenShift - Animation and Page Builder Blocks: by Wpsoul – 8.8.6
Greenshift Smart Code AI: by Wpsoul – 0.2
Greenshift Chart plugin: by Wpsoul – 1.2.2
GreenShift Advanced Animation Addon: by Wpsoul – 3.6.3
Greenshift Query and Meta Addon: by Wpsoul – 4.6
GreenShift Marketing and SEO Addon: by Wpsoul – 0.9.9.9
Greenshift Woocommerce Addon: by Wpsoul – 2.0.9
Health Check & Troubleshooting: by The WordPress.org community – 1.7.0
ThumbPress: by ThumbPress – 5.2.2
Import WP PRO: by James Collings  – 2.10.2
Import WP - WooCommerce Importer Addon: by James Collings  – 2.3.3
Import WP - Custom code: by James Collings  – 0.0.5
Import WP: by James Collings  – 2.13.5
Kadence WooCommerce Email Designer: by Kadence WP – 1.5.12
Loco Translate: by Tim Whitlock – 2.6.9
Meta pixel for WordPress: by Facebook – 3.0.16
Performant Translations: by WordPress Performance Team – 1.2.0
Pinterest for WooCommerce: by WooCommerce – 1.3.24
Price by Formula Calculator for WooCommerce: by Plugify – 3.0.0 (update to version 3.0.9 is available)
React Group XML to Zip: by React Group – 1.0
Really Simple SSL: by Really Simple Plugins – 8.1.3
Regenerate Thumbnails: by Alex Mills (Viper007Bond) – 3.1.6
Skroutz Analytics for WooCommerce: by Skroutz – 1.7.3
Super Speedy Filters: by Dave Hilditch – 5.60
Super Speedy Notices: by Dave Hilditch – 1.0
Super Speedy Search: by Dave Hilditch (WP Intense) – 4.58
SVG Support: by Benbodhi – 2.5.5
Category Order and Taxonomy Terms Order: by Nsp-Code – 1.8.1
The SEO Framework - Extension Manager: by The SEO Framework Team – 2.6.3
User Role Editor: by Vladimir Garagulya – 4.64.2
WC Hide Shipping Methods: by Rynaldo Stoltz – 1.5
WooCommerce Smart COD: by woosmartcod.com – 1.7.2
Challan Pro: by WebAppick – 4.4.1
CTX Feed Pro: by WebAppick Author – 7.3.17
Widget Context: by Kaspars Dambis – 1.3.2
WooCommerce Brands: by WooCommerce – 1.6.68
Checkout Field Editor for WooCommerce (Pro): by ThemeHigh – 3.6.2
Google Analytics for WooCommerce: by WooCommerce – 2.0.7
Product SKU Generator for WooCommerce: by SkyVerge – 2.5.0
Extra Product Options & Add-Ons for WooCommerce: by ThemeComplete – 6.4.5
WooCommerce: by Automattic – 8.7.0 (update to version 8.9.1 is available)
WP Consent API: by RogierLankhorst – 1.0.7
WP Downloader: by Wojtek Szałkiewicz – 2.0
WP Global Site Tag: by Digital Apps – 1.0.7
WP Mail SMTP: by WP Mail SMTP – 4.0.1
WP Paint - WordPress Image Editor: by ZetaMatic – 0.5.3
WP Sheet Editor - Facebook for WooCommerce: by WP Sheet Editor – 1.0.1
WP Sheet Editor - Post Types (Premium): by WP Sheet Editor – 2.25.14-beta.1
WP Sheet Editor - TM Extra Product Options: by WP Sheet Editor – 1.0.1
WP Sheet Editor - Troubleshooting: by WP Sheet Editor – 1.1.0
WP Sheet Editor - WC Product SKU Generator: by WP Sheet Editor – 1.0.1
WP Super Cache: by Automattic – 1.12.1
WPForms Lite: by WPForms – 1.8.8.3
WPMU DEV Dashboard: by WPMU DEV – 4.11.26

### Inactive Plugins (5) ###

Broken Link Checker: by WPMU DEV – 2.2.4
Index WP MySQL For Speed: by Oliver Jones
Rick James – 1.4.18

Query Monitor: by John Blackbourn – 3.16.2
WooCommerce Blocks: by Automattic – 11.7.0
WP File Manager: by mndpsingh287 – 7.2.7

### Dropin Plugins () ###

advanced-cache.php: advanced-cache.php
db.php: Query Monitor Database Class (Drop-in)
object-cache.php: Performance Lab Server Timing Object Cache Drop-In

### Must Use Plugins (4) ###

Health Check Troubleshooting Mode: by  – 1.9.1
Index WP MySQL For Speed Upgrade Filter for mu-plugins.: by  – 1.4.15
mu-check-conflicts.php: by  –
Super Speedy Search Fast Ajax: by Dave Hilditch – 1.05

### Settings ###

API Enabled: –
Force SSL: –
Currency: EUR (€)
Currency Position: right
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to Woo.com: ✔
Enforce Approved Product Download Directories: –
HPOS feature screen enabled: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: ✔

### Logging ###

Enabled: ✔
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 days
Level threshold: –
Log directory size: 8 MB

### WC Pages ###

Shop base: #165323 - /products/
Cart: #418 - /cart/
Checkout: #40 - /checkout/
My account: #421 - /my-account/
Terms and conditions: #5663 - /terms/

### Theme ###

Name: Greenshift
Version: 2.2.5
Author URL: https://greenshiftwp.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: greenshift/woocommerce/myaccount/my-account.php
/home/fshomegr/www/wp-content/plugins/woocommerce/templates/block-notices/error.php
/home/fshomegr/www/wp-content/plugins/woocommerce/templates/block-notices/notice.php
/home/fshomegr/www/wp-content/plugins/woocommerce/templates/block-notices/success.php
greenshift/woocommerce/single-product-reviews.php
greenshift/templates/archive-product.html
greenshift/templates/page-cart.html
greenshift/templates/page-checkout.html
greenshift/templates/product-search-results.html
greenshift/templates/single-product.html

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
product-variation-management
product-virtual-downloadable
product-external-affiliate
product-grouped
product-linked
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: customize-store
minified-js
new-product-management-experience
product-pre-publish-modal
settings
async-product-editor-category-field

Daily Cron: ✔ Next scheduled: 2024-05-23 15:48:13 +03:00
Options: ✔
Notes: 45
Onboarding: completed

### Action Scheduler ###

Complete: 6,984
Oldest: 2024-04-21 19:46:28 +0300
Newest: 2024-05-22 19:12:41 +0300

Failed: 15,540
Oldest: 2022-07-07 20:13:59 +0300
Newest: 2024-05-22 18:55:45 +0300

Pending: 8
Oldest: 2024-05-22 19:50:28 +0300
Newest: 2024-05-27 08:33:42 +0300

### Status report information ###

Generated at: 2024-05-22 19:30:23 +03:00
`

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by kmanijak** [comment]
Hi @gfarma!

Thank you for taking the time to raise this issue(s)!

> At one of the recent updates the ability to duplicate the pagination block is not availabe ( I use 8.7.0 and I it is not possible)

I was able to reproduce this issue and already prepared a fix: https://github.com/woocommerce/woocommerce/pull/47749. The PR is under review and will land in WooCommerce 9.0.

> Also after the update 8.7.0 if you add a another block inside the product collection query block, the interactivity pagination with back to top is not working. (even if you add a simple text block from a FSE builder - not core).

This is intended change as per https://github.com/woocommerce/woocommerce/pull/45435. At the moment, we're not able to determine if the 3rd party blocks are compatible with Interactivity API or not and to make sure all blocks work correctly, we disable client-side navigation if 3rd party blocks are detected.

There's no API in WordPress 6.5 allowing declaring Interactivity compatibility by:

```
"supports": {
    "interactivity": true
},
```

([more info here](https://make.wordpress.org/core/2024/03/04/interactivity-api-dev-note/)). We'll be able to use this API to determine compatibility but only once we bump minimum required version of WordPress to 6.5.

I hope that answers your concerns.
